### PR TITLE
all: fix docstrings

### PR DESCRIPTION
**Pull request by availhang**

### CODE REVIEW

File: common/math/big.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ func (i *HexOrDecimal256) MarshalText() ([]byte, error) {
 	return []byte(fmt.Sprintf("%#x", (*big.Int)(i))), nil
 }
 
-// Decimal256 unmarshals big.Int as a decimal string. When unmarshalling,
```
Comment (jwasinger) [inline common/math/big.go:81]: This change is wrong and the line should be kept as-is.

Comment (availhang) [reply common/math/big.go]: Thanks!
Reverted

File: beacon/light/head_tracker.go [hunk 2]
```diff
@@ -56,7 +56,7 @@ func (h *HeadTracker) ValidatedHead() (types.SignedHeader, bool) {
 	return h.signedHead, h.hasSignedHead
 }
 
-// ValidatedHead returns the latest validated head.
```
Comment (jwasinger) [inline beacon/light/head_tracker.go:59]: The previous method description was correct.

Comment (availhang) [reply beacon/light/head_tracker.go]: Thanks!
Modified.

**Review by rjl493456442 — APPROVED** [review]

**Pull request merged**